### ISSUE TITLE
Reply on deleted email disappears 

### ISSUE THREAD

**Issue by joris85** [freescout-helpdesk/freescout#585]
Hi guys,

I had an email of a college.
To keep my history a bit clean i deleted this email.
Then my college replied on that e-mail and i never saw that reaction.

I would think that like a closed one it would reopen again or start as a new email because it just disappeared right now.
That is not helping to keep my database clean and delete emails, that is dangerous to delete :-D
Is this the way it will work, will there come a setting how to work with this situation or is this a bug?
Love to hear what your thoughts are on a situation like this!

**Comment by freescout-helpdesk** [comment]
Have you deleted the conversation forever or moved it to the Deleted folder?

**Comment by Speedy059** [comment]
This is a confirmed issue. A colleague of mine reported this to me. This is how to reproduce:

1) Send an email to freescout inbox
2) In Freescout, respond to the email and then Delete it. (Deleting it first before responding has no issues)
3) Send reply to the email you got from Freescout.
4) Go back into freescout and the response you sent to Freescout is in the Deleted folder now.

I noticed this happens only to emails that have been responded to, and then deleted. We found responses in the deleted folder. It's almost like it's tagging the emails as spam now and then auto deletes the response.

If you send an email that is NOT response from freescout, it gets imported just fine and isn't auto-deleted. This is only happening to conversation responses.
